data/assets/scene/solarsystem/planets/jupiter/minor/ananke_group.asset

```diff
@@ -207,22 +207,61 @@ local anankeGroup = {
   }
 }
 
+--Generate all moon labels
```

Comment:
I'd usually have a space between the `--` and the comment to make it easier to read, but that's personal preference

Reply:
Fixed in: [055eab06540015eba3138c73cb4cb4bf6f36b5ad](https://github.com/OpenSpace/OpenSpace/commit/055eab06540015eba3138c73cb4cb4bf6f36b5ad)